src/locales/en/translation.json

```diff
@@ -1756,36 +1757,26 @@
     "limitOrderHistory": {
       "limitPriceHelper": "Final execution price of Limit Orders is impacted by cost of relayer fees and slippage at the time of execution. For further details see the <0>wiki</0>.",
       "filledPriceHelper": "Final execution price of Limit Orders is impacted by cost of relayer fees and slippage at the time of execution, therefore this will not match the defined limit price exactly. For further details see the <0>wiki</0>."
-    },
-    "notificationSettingsDialog": {
```

Comment:
Please remove this section from any other localisation json files that it might be present in (e.g. pt_br)